### DISCUSSION TITLE
Doesn't download files

### DISCUSSION THREAD

**Discussion by DEFALTtttttt** [Ylianst/MeshCentral#6400] [Issues]
![image](https://github.com/user-attachments/assets/9f356f07-e8d0-4dba-a508-9b6fc6c35f10)
When I try to download any file from a remote computer, I get the error "Check your Internet connection"
server version 1.1.27
WIn 10

**Comment by si458** [comment]
Another user had this problem 

Make sure u don't have any like browser extensions turned on that download files for u

**Reply by DEFALTtttttt** [reply]
There are no extensions, there is a link to a thread with the same problem. And how will it be solved?

**Reply by si458** [reply]
Without seeing your config.json i don't no what the issue is or even how to replicate the issue

**Reply by DEFALTtttttt** [reply]
nothing changed in json, and before that everything worked, it stopped with the server update

**Reply by si458** [reply]
Again without seeing ur config.json and what node/os ur running etc we can't replicate ur issue

**Reply by DEFALTtttttt** [reply]
![image](https://github.com/user-attachments/assets/5ac208a7-3fb9-4034-a2ee-9b554f3b3956)
